Source/WebKit/Platform/IPC/StreamConnectionEncoder.h

```diff
@@ -77,7 +76,7 @@ class StreamConnectionEncoder final {
     bool encodeObject(const T& object)
     {
         static_assert(std::is_trivially_copyable_v<T>);
-        return encodeSpan(std::span(std::addressof(object), 1));
+        return encodeSpan(unsafeMakeSpan(std::addressof(object), 1));
```

Comment:
Should we put a wrapper somewhere that safely makes a span for a single object we have a reference to? Doesn’t seem at all unsafe!

Reply:
```
template<typename T>
std::span<const T> asSingleElementSpan(const T& object)
{
    return unsafeMakeSpan(std::addressof(object), 1);
}
```

Or do you have a suggestion for a better name?

Reply:
I like the name. Although I would just call it `singleElementSpan`.

Reply:
Ok.